lib/internal/process/warning.js

```diff
@@ -28,11 +29,16 @@ function setupProcessWarnings() {
 
   // process.emitWarning(error)
   // process.emitWarning(str[, name][, ctor])
-  process.emitWarning = function(warning, name, ctor) {
+  // process.emitWarning(str[, name][, ctor][, warned])
+  process.emitWarning = function(warning, name, ctor, warned) {
     if (typeof name === 'function') {
       ctor = name;
       name = 'Warning';
     }
+    if (typeof arguments[arguments.length - 1] === 'boolean') {
+      warned = arguments[arguments.length - 1];
+    }
```

Comment:
Ah... yeah, you're right. Sorry about that. Hmm... ok, I'm good with this like this then. Might revisit it later then to see if there's a better way.